private/TupleManipulation.h

```diff
@@ -6,10 +6,10 @@ namespace TupleManipulation {
    struct sequence { };
 
    template<int N, int... Integers>
-   struct gen_sequence : gen_sequence<N - 1, N - 1, Integers...> { };
+   struct gen_sequence : public gen_sequence<N - 1, N - 1, Integers...> { };
 
    template<int... Integers>
-   struct gen_sequence<0, Integers...> : sequence<Integers...> { };
+   struct gen_sequence<0, Integers...> : public sequence<Integers...> { };
```

Comment:
ooo weird

Reply:
IAR was complaining about the lack of explicit access permission